### ISSUE TITLE
Light network repl mode for slow remote repl work

### ISSUE THREAD

**Issue by CmdrDats** [cursiveclojure/cursive#1001]
Often, I need to repl connect to a remote server where the connection isn't ideal - would it be possible to have a limited mode that you can enable in the remote repl setup that doesn't make any extra autocompletion/environment inspection calls so that things go faster. Essentially allowing you to choose to have that tradeoff

**Comment by cursiveclojure** [comment]
Yes, this sounds like a good idea. In that case I can offer completions etc from the open project, so as long as you're connecting to a remote copy of the project you have open the completion should be pretty good.
